tests/test_verification.py

```diff
@@ -0,0 +1,154 @@
+import unittest
+
+from rflx.model import ModelError
+from rflx.parser import Parser
+
+
+class TestVerification(unittest.TestCase):  # pylint: disable=too-many-public-methods
+    def setUp(self) -> None:
+        self.maxDiff = None  # pylint: disable=invalid-name
+
+    def assert_parse_exception_string(self, string: str, regex: str) -> None:
+        with self.assertRaisesRegex(ModelError, regex):
+            Parser().parse_string(string)
+
+    def test_exclusive_valid(self) -> None:  # pylint: disable=no-self-use
+        parser = Parser()
+        parser.parse_string(
+            """
+            package Foo is
+                type Element is range 0..2**32-1 with Size => 32;
+                type Bar is
+                message
```

Comment:
Strange indentation of `message`. It should be either indented (like `record` in Ada; cf. Ada Style Guide) or at the same line as `is`.

Reply:
Noooooooo! I insist on formatting that differently!1! Just kidding - addressed in f067aa0 ;-)